readme.md

```diff
@@ -1,7 +1,7 @@
 # Product cards
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards-en/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards-en/report/html_report/)
+- [DEMO LINK](https://proznerka.github.io/layout_product-cards-en/)
+- [TEST REPORT LINK](https://proznerka.github.io/layout_product-cards-en/report/html_report/)
```

Comment:
For some reason your test link and demo links do not open please re-deploy code.